### PR TITLE
[UI]Added modal for updating in grid view of applications 

### PR DESCRIPTION
**Pull request by partik03**
**Notes for Reviewers**

This PR fixes #
#6798 

**Before**

https://user-images.githubusercontent.com/99184676/209821525-b5cf4dc9-8b67-4b85-a2c4-40f73e26753a.mp4

**After**

https://user-images.githubusercontent.com/99184676/209821549-6a635fc9-5ee4-4a3f-ac5c-64c9a43bab29.mp4

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/MesheryApplications.js [hunk 1]
```diff
@@ -519,6 +524,17 @@ function MesheryApplications({
         () => {
           console.log("ApplicationFile API", `/api/application/${source_type}`);
           updateProgress({ showProgress : false });
+          enqueueSnackbar(`"${name}" Application Updated Successfully`, {
```
Comment (Chadha93) [inline ui/components/MesheryApplications.js:527]:
```suggestion
          enqueueSnackbar(`"${name}" Application updated`, {
```

**Comment by Chadha93** [comment]
@partik03 Some good work here, could you please add this to is filters and designs page as well?

**Comment by Chadha93** [comment]
Also if you want to discuss please add this as an agenda item for today's dev call https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#

**Comment by partik03** [comment]
> @partik03 Some good work here, could you please add this to is filters and designs page as well?

Yes I will soon make a pr for them as well

**Comment by partik03** [comment]
> Also if you want to discuss please add this as an agenda item for today's dev call https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#

I won't be available today,I am a bit busy.Really sorry for that

**Comment by leecalcote** [comment]
If a user clicks Save, then Meshery should perform a save operation directly. There's no need for user confirmation of this action. Unlike confirmation for a delete action, saves can be undone.

**Comment by partik03** [comment]
> If a user clicks Save, then Meshery should perform a save operation directly. There's no need for user confirmation of this action. Unlike confirmation for a delete action, saves can be undone.

But the user can accidentally change the application,I myself changed it accidentally once so thought of implementing this :)

**Pull request closed without merging**